### PR TITLE
#174797503 Configure BareFoot Nomad to use PostgreSQL

### PR DESCRIPTION
**Pull request by florisng**
#### What does this PR do?
Configure BareFoot Nomad to use PostgreSQL as DB Engine

#### Description of Task to be completed?
Connect the app to postegreSQL

#### How should this be manually tested?
1. Clone the repo git clone `https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
2. Switch your working directory `cd script-heroes-bn-backend`
3. Checkout the branch `git checkout ch-configure-postgresql-#174797503`
4. Download & Install postgreSQL `https://get.enterprisedb.com/postgresql/postgresql-12.4-1-windows-x64.exe`
5. Create database called `database` with password `password`
4. Run `npm run dev`

#### What are the relevant pivotal tracker stories?
[#174797503](https://www.pivotaltracker.com/story/show/174797503)

### CODE REVIEW

**Review by valens10 — CHANGES_REQUESTED** [review]
Migrate to ES6

**Review by Habinezajanvier — APPROVED** [review]
Ok good

**Review by valens10 — APPROVED** [review]

File: src/services/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (kipropbrian) [inline src/services/index.js]: Kindly remove blank/empty files

**Comment by kipropbrian** [comment]
These instruction are really poor

How should this be manually tested?

    Clone this repo
    Checkout your branch
    Run npm run dev
    You should see a message confirming that the server is listening to port 3000 and the connexion to the database.



Please put a more detailed set of instruction with the actual commands I need to run in order for this to work

**Pull request merged**